lmdeploy/pytorch/models/deepseek_v2.py

```diff
@@ -1169,6 +1169,12 @@ def prepare_inputs_for_generation(
         position_ids = context.position_ids
         attn_metadata = context.attn_metadata
 
+        from dlblas.layers.moe.token_dispatcher import DeepEPBuffer, DeepEPMode
```

Comment:
move this to backend
https://github.com/InternLM/lmdeploy/blob/f63730deb1be7684fb20c7c42a4e14e0fabd5245/lmdeploy/pytorch/backends/cuda/moe.py#L450

You can get context with context manager
https://github.com/InternLM/lmdeploy/blob/f63730deb1be7684fb20c7c42a4e14e0fabd5245/lmdeploy/pytorch/model_inputs.py#L531